src/components/Icon/Icon.module.scss

```diff
@@ -3,7 +3,7 @@
   .iconWrapper {
     width: 36px;
     height: 36px;
-    background-color: $dark-grey--7;
+    background-color: $dark-grey--6;
```

Comment:
Having the 6 in here is pretty generic naming. Is there any better naming solution for that grey?